understanding/22/dragging-movements.html

```diff
@@ -62,7 +62,7 @@ <h3>Alternatives for dragging movements on the same page</h3>
       <section>
          <h3>Distinguishing dragging movements from path-based pointer gestures</h3>
       
-         <p>Dragging movements covered in this Success Criterion are pointer interactions where only the start- and endpoints matter. Once the pointer engages with a target, the direction of the dragging movement does not factor into the interaction until the pointer disengages the target. Since the dragging movement does not have an intermediate point, the dragging movement can go in any direction. Path-based gestures are covered in Success Criterion 2.5.1 Pointer Gestures. For more details, refer to <a href="https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html">Understanding Success Criterion 2.5.1 Pointer Gestures</a></p> 
+         <p>Dragging movements covered in this Success Criterion are pointer interactions where only the start- and endpoints matter. Once the pointer engages with a target, the direction of the dragging movement does not factor into the interaction until the pointer disengages the target. Since the dragging movement does not have an intermediate point, the dragging movement can go in any direction. Path-based gestures are covered in <a href="pointer-gestures">Success Criterion 2.5.1 Pointer Gestures</a>.</p> 
```

Comment:
```suggestion
         <p>Dragging movements covered in this Success Criterion are pointer interactions where only the start- and endpoints matter. Once the pointer engages with a target, the direction of the dragging movement does not factor into the interaction until the pointer disengages the target. Since the dragging movement does not have an intermediate point, the dragging movement can go in any direction. Path-based gestures are covered in <a href="pointer-gestures.html">Success Criterion 2.5.1 Pointer Gestures</a>.</p> 
```

Reply:
Minor thing, I think it would work when live, but with the .html included it also works when building locally.